server/activityItem/fetch.ts

```diff
@@ -121,9 +122,7 @@ const applyFiltersToWhereQuery = (whereQuery: any, filters: ActivityFilter[]) =>
 	return whereQuery;
 };
 
-const fetchActivityItemModels = async (
-	options: Required<FetchActivityItemsOptions>,
-): Promise<types.ActivityItem[]> => {
+const fetchActivityItemModels = async (options: Required<FetchActivityItemsOptions>) => {
```

Comment:
In a lot of cases, explicit return or parameter types are no longer necessary, in which case i removed them.